release-schema.json

```
@@ -48,18 +65,39 @@
             "null"
           ],
           "minLength": 1
+        },
+        "competitiveMaximumPercentage": {
+          "title": "Maximum percentage of contract value for competitive subcontracting",
+          "description": "The maximum percentage of the contract value that the procuring entity requires the supplier to subcontract to third parties through a competitive procedure.",
```

Applying style guide. Changing verb to make meaning clearer (the maximum is there to avoid, e.g. 90% subcontracted, which means the bidder is basically just extracting rents)

```suggestion
          "description": "The maximum percentage of the contract value that the buyer or procuring entity allows the supplier to subcontract to third parties through a competitive procedure.",
```